doc/overview/project_capabilities.md

```diff
@@ -12,6 +12,15 @@ known to declare the capability.
 | VisualC | Project may contain or compile C++ source files.
 | CSharp | Project may contain or compile C# source files.
 | VB | Project may contain or compile VB source files.
+| JavaScript | Project contains and handles JavaScript source files.
```

Comment:
I think that this is really a mark other than a declaration of capability. How about we remove it and not document it?